src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -384,6 +384,62 @@ public void SendPacketsElement_FileLargeCount_Throws(SocketImplementationType ty
             SendPackets(type, new SendPacketsElement(TestFileName, 5, 10000), SocketError.InvalidArgument, 0);
         }
 
+        [Theory]
+        [InlineData(SocketImplementationType.APM)]
+        [InlineData(SocketImplementationType.Async)]
```

Comment:
I think you'll need to add a SkipOnTargetFramework attribute to skip this on netfx, since it has the same bug and isn't getting your fix.  The lack of that is why at least one of the CI legs is failing. e.g.
https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.Runtime/tests/System/ValueTypeTests.cs#L35

Reply:
Ah yes, good call. I was wrecking my brain to figure out the NETFX fail. What about the UWP build, where the new test also fails as if the fix is not there, is that a similar story? I'm not sure exactly what that is building.

If so, would it make more sense to move this test to `SendPacketsAsync.netcoreapp.cs` instead of adding the attribute?

Reply:
> What about the UWP build, where the new test also fails as if the fix is not there, is that a similar story? 

Hmm, I wouldn't expect that to fail, as it should be using this implementation. It also doesn't look to me like it failed because of this, but maybe I'm missing it?

Reply:
> It also doesn't look to me like it failed because of this, but maybe I'm missing it?

Looks like you're right, my bad. I'll just add the attribute and hopefully the UWP build will work itself out...